python/README.md

```diff
@@ -0,0 +1,8 @@
+pb2 files can be regenerated by running:
+
+```
+python3 -m grpc_tools.protoc -I../proto/ --python_out=gubernator/ --pyi_out=gubernator/ --grpc_python_out=gubernator/ --mypy_grpc_out=gubernator/ ../proto/*.proto
```

Comment:
Can you move this stuff to a bash script in `proto` directory?  And include a `pip install` for the tools.

And preferably we shouldn't need post-processing with `sed` commands.  Looks like it's a matter of getting CWD or input paths right to fix that.

Reply:
> And preferably we shouldn't need post-processing with `sed` commands. Looks like it's a matter of getting CWD or input paths right to fix that.

Literally spent a couple of hours looking at this. In summary, it appears that Google have no interest in supporting modules  which would require relative imports. It might be possible to get something to work by changing the proto directory, but I figured that might break things elsewhere. Feel free to have a go yourself though.

Reply:
As you can probably tell, we don't use this Python functionality anymore.  At least, not for Python 3.  I'm thinking the entire Python client needs to be reimplemented from protos and disregard any consideration for legacy requirements.

I committed some changes to [my branch](https://github.com/mailgun/gubernator/tree/Baliedge/update-python-proto) for a `genproto.sh` script that already exists but needed some cleaning up.  It does generate the code for Go and Python ok.  There's some old unit tests in `python/tests`, but no idea how compatible that is to now current code.  It fails, as you might expect.

Reply:
The `sed` calls are not related to legacy (in fact, it apparently worked on Python 2 without rewriting the imports). It is because Google apparently just dumps all (thousands) of their files in a top-level directory that is on their PYTHONPATH.

For an installable package, we can't expect users to change their PYTHONPATH to point to the package's source code (I know that sound ridiculous, but I literally don't understand why the protobuf maintainers in that thread don't appear to understand that). So, `sed` rewrites the imports to import from the package (i.e. it could be done either as `from gubernator` or `from .`).

To test, run `pip install .` in your python package, then (ensuring you're not in the source directory) open python and `import gubernator`. Without the `sed` changes you'll get an import error.

I'll go through the other changes next week.